project_templates/fastapi_safir_app/example/pyproject.toml

```diff
@@ -53,3 +53,10 @@ include_trailing_comma = true
 multi_line_output = 3
 known_first_party = ["example", "tests"]
 skip = ["docs/conf.py"]
+
+[tool.pytest.ini_options]
+asyncio_mode = "strict"
+python_files = [
```

Comment:
Ah, I was wondering about this after seeing it in gafaelfawr.

I'm kind of confused why this setting provides "verbose diffs on failed asserts" rather than just ensuring that tests are detected properly. This is compatible with the `tests/support/` pattern for mocks? Or indeed, is it because of this pattern?